### PR TITLE
Adding new couchdb node vocms0745 for k8s cluster

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
@amaltaro , @vkuznet 
Could you please review this PR for new couchdb node for k8s cluster.

Best regards
M Imran

File: admin/deploy [hunk 1]
```diff
@@ -73,8 +73,8 @@ deploy_admin_post()
   fi
 
   case $host:$root in
-    vocms0117:/data/srv | vocms0127:/data/srv | vocms0134:/data/srv | vocms0734:/data/srv | vocms0135:/data/srv | \
```
Comment (vkuznet) [inline admin/deploy:76]: I see that there are changes to other hostnames which is not related to vocms0745. I have no idea if they are required, but this PR should provide clear description of what is added (my understanding it is only vocms0745) and what is removed (I see several nodes, like vocms0127, vocms0134, vocms0160).

File: crabserver/deploy [hunk 2]
```diff
@@ -38,7 +38,7 @@ deploy_crabserver_sw()
 
 deploy_crabserver_post()
 {
-  case $host in vocms013[89] | vocms073[89] | vocms0143 | vocms074[01234] | vocms0307 | vocms0318 ) disable ;; * ) enable ;; esac
```
Comment (vkuznet) [inline crabserver/deploy:41]: here are more nodes which are removed.

File: admin/deploy (hunk 1 above)
Comment (muhammadimranfarooqi) [reply admin/deploy:76]: These nodes do not exist any more in cmsweb.

Comment (muhammadimranfarooqi) [reply admin/deploy:76]: This is the list of nodes that I noticed during creating this PR. These nodes don't exist anymore in cmsweb. So, I thought to remove them as well.

vocms0126
vocms0127
vocms0128
vocms0134
vocms0138
vocms0139
vocms0160
vocms0306
vocms0307
vocms0318

**Comment by amaltaro** [comment]
Imran, before we go down this road. Have you heard anything back from @khurtado Kenyi on the kubernetes tests? Depending on how that goes, we will not have the need to keep a VM serving couchdb requests for the k8s clusters (especially because plugging multiple clusters will likely end badly)

File: admin/deploy (hunk 1 above)
Comment (vkuznet) [reply admin/deploy:76]: in this case, please do not mix different things in single PR in a future, otherwise it is hard to review.

**Comment by muhammadimranfarooqi** [comment]
> Imran, before we go down this road. Have you heard anything back from @khurtado Kenyi on the kubernetes tests? Depending on how that goes, we will not have the need to keep a VM serving couchdb requests for the k8s clusters (especially because plugging multiple clusters will likely end badly)

@amaltaro 

I did not hear from @khurtado  on k8s tests. Furhtermore, this VM is only for testing in k8s testbed cluster not for multiple clusters.

File: base/deploy [hunk 3]
```diff
@@ -16,6 +16,6 @@ deploy_base_sw()
 
 deploy_base_post()
 {
-  case $host in vocms013[89] | vocms073[89] | vocms074[034]) disable ;; * ) enable ;; esac
+  case $host in vocms013[89] | vocms073[89] | vocms074[0345]) disable ;; * ) enable ;; esac
```
Comment (amaltaro) [inline base/deploy:19]: I don't see 0731 here, so I would say we do not need to add 0745. But honestly, I don't know what this base/deploy does...

Comment (muhammadimranfarooqi) [reply base/deploy:19]: I dont know about this script.

**Comment by vkuznet** [comment]
in general changes are fine with me, but I'll leave it up to you and Alan to decide.

**Comment by amaltaro** [comment]
Imran, as Valentin said, it's a bit hard to review it because it has other changes in as well.
However, it looks good to me. Thanks

**Pull request closed without merging**